db.go

```diff
@@ -192,6 +192,67 @@ type (
 	BucketMetasIdx map[string]*BucketMeta
 )
 
+func (e Entries) Len() int { return len(e) }
```

Comment:
Would suggest this part of code move to another seperated file so that make db.go focus its own logic.

Reply:
I moved `Entries` to `entry.go`